crates/aggkit-prover-config/tests/fixtures/validate_config/max_rpc_request_size.toml

```diff
@@ -0,0 +1,2 @@
+[rpc]
```

Comment:
We should also keep to common test configurations on one place. Add only specified `aggkit-prover-config` and `agglayer-prover-config` to their respective folders.